plugin/net/cni.go

```diff
@@ -203,15 +208,18 @@ func (c *CNIPlugin) CmdDel(args *skel.CmdArgs) error {
 		err = ipam.ExecDel(conf.IPAM.Type, args.StdinData)
 	}
 	// Hack - don't know how we should detect this situation properly
-	if args.Netns == "" && strings.Contains(err.Error(), "no addresses") {
-		err = nil
-	}
-	if err != nil {
-		return fmt.Errorf("unable to release IP address: %s", err)
+	if args.Netns != "" || !strings.Contains(err.Error(), "no addresses") {
+		logToKubelet(fmt.Errorf("unable to release IP address: %s", err))
 	}
 	return nil
 }
 
+// Standard error is inherited from the caller (typically Kubelet) hence
+// provided errors should eventually appear in the caller's log files.
+func logToKubelet(err error) {
+	fmt.Fprintln(os.Stderr, err)
```

Comment:
Sure, will go for `weave-cni:` as a prefix then.
See: https://play.golang.org/p/JcRUOCC5Fq